### ISSUE TITLE
Not working with Ionic

### ISSUE THREAD

**Issue by joseserrano15** [Brakebein/ngx-tagify#39]
Hi, I'm using Ionic + Angular and the library doesn't work, it show very extrange (see image). I'm trying to use it with reactive forms.

```html
<tagify formControlName="email"></tagify>
```

```ts
newClintForm = new FormGroup({
  identificationType: new FormControl('', Validators.required),
  indentification: new FormControl('', Validators.required),
  name: new FormControl('', Validators.required),
  socialReazon: new FormControl('', Validators.required),
  address: new FormControl('', Validators.required),
  email: new FormControl([], [Validators.email, Validators.required]),
  phone: new FormControl('', Validators.required)
})
```

![image](https://user-images.githubusercontent.com/68348356/207727917-6a2e13fc-e678-47a7-80c9-8bf1e07d3adc.png)

**Comment by Brakebein** [comment]
The standard validators won't work with this kind of data. Try to remove the validators and look if it then behaves correctly.

You can pass a specific pattern to the tagify settings to validate the input:

```ts
const EMAIL_REGEXP =
    /^(?=.{1,254}$)(?=.{1,64}@)[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+)*@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;

this.tagifySettings: TagifySettings = {
  pattern: EMAIL_REGEXP
};
```
```html
<tagify formControlName="email" [settings]="tagifySettings"></tagify>
```

**Comment by joseserrano15** [comment]
I tried as you mentioned but didn't work, I removed the validators and I used the settings property, but in this way if I write in the input inmediatly the value is erased.  I tried even using ngModel way but I had the same problem (it shows very strange like the image).

I attach an image of the console with the ngmodel mode.

![image](https://user-images.githubusercontent.com/68348356/207945559-9a4a37ef-bcac-4a62-b59c-118ebc390daa.png)

![image](https://user-images.githubusercontent.com/68348356/207945656-05995bad-8445-482f-a7a1-6d5af9278d7e.png)
